app/views/users/show.html.erb

```diff
@@ -0,0 +1,4 @@
+<p id="notice"><%= notice %></p>
+
+<%= link_to 'Edit', edit_user_path(@user) %> |
```

Comment:
Following up on my previous comment since I realized I might not have been clear: If we can't update users yet, we shouldn't have an edit link at this point.

Reply:
This file was also deleted